packages/server/lib/controllers/appAuth.controller.ts

```diff
@@ -27,8 +27,16 @@ class AppAuthController {
     async connect(req: Request, res: Response, _next: NextFunction) {
         const installation_id = req.query['installation_id'] as string | undefined;
         const state = req.query['state'] as string;
+        const action = req.query['setup_action'] as string;
 
-        if (!state || !installation_id) {
```

Comment:
In this instance the request comes from email so after they approve we need to send them somewhere. Just send them back where they came from.